css/common.css

```diff
@@ -0,0 +1,254 @@
+     *,
+     *::before,
+     *::after {
+         box-sizing: border-box;
+         margin: 0;
+         padding: 0;
+         border: 0;
+     }
+     
+      :root {
+         /* ### Colors */
+         --clr-dark-beige: #E6DACD;
+         --clr-light-beige: #F4ECE6;
+         --clr-royal-blue: #0150FD;
+         --clr-black: #000;
+         --clr-white: #FFF;
+         /* ### Fonts-Family */
+         --font-poppins: 'Poppins', sans-serif;
+         --font-quicksand: 'Quicksand', sans-serif;
+         /* ### Fonts Size */
+         --font-size--14px: 0.875rem;
+         --font-size--15px: 0.9375rem;
+         --font-size--16px: 1rem;
+         --font-size--17px: 1.0625rem;
+         --font-size--19px: 1.1875rem;
+         --font-size--20px: 1.25rem;
+         --font-size--21px: 1.3125rem;
+         --font-size--40px: 2.5rem;
+         /* ### Fonts Weight */
+         --font-weight--400: 400;
+         --font-weight--600: 600;
+         --font-weight--700: 700;
```

Comment:
nice use of CSS variables! I think more meaningful names for font-weight and font-size variables can include the use cases for the variables, for example
```
--font-size--body: 0.875rem;
--font-size--heading: 2.5rem;
--font-size--sub-heading: 1.3125rem;
```